### DISCUSSION TITLE
Failed to train model

### DISCUSSION THREAD

**Discussion by Jaxyk** [smy20011/dreambooth-gui#97] [Q&A]
![Image 3](https://github.com/smy20011/dreambooth-gui/assets/120110463/0e2442b5-b9e2-46e5-8801-b16b8db88744)
Error at the beginning of work

**Comment by msandroid** [comment]
me too

**Comment by ajamespaterson** [comment]
I have the same error. Any solution to this?

**Comment by ajamespaterson** [comment]
Just wanted to update: I "fixed" this by reverting the model to the default CompVis/stable-diffusion-v1-4 one. Seems the program didn't like when I attempted to use Realistic_Vision_V2.0.ckpt.
